lint-workflow/lint.py

```diff
@@ -167,19 +168,26 @@ def lint(filename):
         print()
 
 
-def main():
+def main(input_args=None):
+
+    # Pull the arguments from the command line
+    if not input_args:
+        input_args = sys.argv[1:]
 
     # Read arguments from command line.
     parser = argparse.ArgumentParser()
     parser.add_argument("input", help="file or directory input")
-    args = parser.parse_args()
+    args = parser.parse_args(input_args)
 
     # Set up list for files to lint.
     input_files = []
 
     # Check if argument is file, then append to input files.
     if os.path.isfile(args.input):
         input_files.append(args.input)
+    # Check if argument contains multiple files.
+    elif len(args.input.split()) > 1:
+        input_files.extend(args.input.split())
```

Comment:
Should we confirm that each split string is a file with `os.path.isfile`?  I'm worried about accidentally specifying multiple directories of a mix of directories and files.  We don't have to support those options, but just some extra checking to make sure it doesn't fail.